libs/community/langchain_community/vectorstores/opensearch_vector_search.py

```diff
@@ -732,7 +740,47 @@ async def adelete(
         return not any(
             item.get("delete", {}).get("error") for item in response["items"]
         )
-
+    
+    def create_search_pipelines(
+        self,
+        pipeline_name: str,
+        keyword_weight: float = 0.7,
+        vector_weight: float = 0.3,
+    )-> requests.Response:
+        path=f"_search/pipeline/{pipeline_name}"
+        auth = self.http_auth
+        url = f"{self.opensearch_url}/{path}"
+
+        payload = {
+        "description": "Post processor for hybrid search",
+        "phase_results_processors": [
+            {
+            "normalization-processor": {
+                "normalization": {
+                "technique": "min_max"
+                },
+                "combination": {
+                "technique": "arithmetic_mean",
+                "parameters": {
+                    "weights": [
+                    keyword_weight,
+                    vector_weight
+                    ]
+                }
+                }
+            }
+            }
+        ]
+        }
+        try:
+            response = requests.put(url, auth=auth, json=payload, verify=False)
```

Comment:
verify=False is a security risk

Reply:
I'm actually using OpenSearch locally. that's why I set it to "False".

Reply:
I understood, that for AOSS we need to be careful.